tailwind.base.config.js

```diff
@@ -72,6 +72,9 @@ module.exports = {
           },
         ],
       },
+      opacity: {
+        45: 0.45,
+      },
```

Comment:
Not a fan of adding variants here, I mean... what if we need `opacity: 55` now? sometimes CSS values have to be tweaked quite precisely to have a good result. And I don't see the added value of having a predefined tailwind class, in the end the generated CSS will have

```css
opacity-45 {
  opacity: 0.45;
}
```

instead of

```css
opacity-[0.45] {
  opacity: 0.45;
}
```

Not a huge difference I think? Or am I missing something?

Reply:
Yes but the config also has various font sizes for example, and in time it might keep growing. Tailwind now offers the possibility to write custom values in brackets which I think it's awesome; if it wasn't possible I would probably use inline css instead, because sometimes you just need to provide a very specific value.

Consistency is important but in my experience CSS is really about tweaking values to get the result just right. Also sometimes UX designers build pixel-perfect designs that does not necessarily rely on standardized margins, paddings, sizes etc. UX > technical considerations :slightly_smiling_face: